buildenv/jenkins/variables/defaults.yml

```diff
@@ -411,6 +411,30 @@ x86-64_mac:
     11:
       - special.system
 #========================================#
+# Mac M1 Aarch64
+#========================================#
+aarch64_mac:
+  extends: ['boot_jdk_default']
+  boot_jdk:
+    arch: 'aarch64'
+    os: 'mac'
+    dir_strip:
+      all: '3'
+  release:
+    all: 'macosx-aarch64-server-release'
+    8: 'macosx-x86_64-normal-server-release'
+    11: 'macosx-x86_64-normal-server-release'
```

Comment:
I think these "8" and "11" lines should be `aarch64` instead of `x86_64`.

Reply:
good catch